apps/test/unit/code-studio/components/header/SignInCalloutTest.js

```diff
@@ -6,26 +6,27 @@ import i18n from '@cdo/locale';
 import sinon from 'sinon';
 import cookies from 'js-cookie';
 
-const wrapper = shallow(<SignInCalloutWrapper />);
-const parentWrapper = mount(<SignInCalloutWrapper />);
-
 describe('ViewPopup', () => {
   it('displays the correct background darkness', () => {
+    const wrapper = shallow(<SignInCalloutWrapper />);
     wrapper.setState({hideCallout: false});
     expect(wrapper.html().includes('opacity:0.5')).to.be.true;
   });
 
   it('shows the correct header', () => {
+    const wrapper = shallow(<SignInCalloutWrapper />);
     expect(wrapper.html().includes(i18n.notSignedInHeader())).to.be.true;
   });
 
   it('shows the correct image', () => {
+    const wrapper = shallow(<SignInCalloutWrapper />);
     expect(wrapper.html().includes('user-not-signed-in.png')).to.be.true;
   });
 });
 
 describe('Check popup does not appear when flag is set', () => {
   it('does not obscure the current view', () => {
+    const wrapper = shallow(<SignInCalloutWrapper />);
     wrapper.setState({hideCallout: true});
     expect(wrapper.html() === null).to.be.true;
```

Comment:
Sorry, just noticed this, but I think you can write this as `expect(...).to.be.null;`  (Up to you if you want to fix.)

Reply:
Thanks for this! Updated.